core/templates/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -69,6 +69,8 @@ export class StateEditorService {
   stateHintsEditorInitialised: boolean = false;
   stateSolutionEditorInitialised: boolean = false;
   stateEditorDirectiveInitialised: boolean = false;
+  _stateEditorInitializedEventEmitter = new EventEmitter();
+  _stateEditorDirectiveInitializedEventEmitter = new EventEmitter();
```

Comment:
Since this is inside a ts class you can add the `private modifier`

Reply:
Done, and moved to top